concent_api/concent_api/settings/base.py

```diff
@@ -290,3 +291,6 @@
 
 # URL format: 'protocol://<user>:<password>@<hostname>:<port>/<virtual host>'
 # CELERY_BROKER_URL = ''
+
+# Debug setting for adding stack traces in HTTP500 responses
+# DEBUG_INFO_IN_ERROR_RESPONSES = ''
```

Comment:
This is not a string. Please remove the misleading "default". It's fine to leave it as:

``` python
# Debug setting for adding stack traces in HTTP500 responses
#DEBUG_INFO_IN_ERROR_RESPONSES =
```

Reply:
done